### PR TITLE
LPS 86217 Create Reset Password Feature

### PR DESCRIPTION
**Pull request by camisLuisa**

### CODE REVIEW

File: android/forms-screenlet-demo/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
 	<string name="app_name">TGF</string>
+	<string name="continue_">CONTINUE</string>
+	<string name="continue_insurance_quote_form">Continue Insurance Quote form?</string>
+	<string name="could_not_load_form">Could not load Form</string>
 	<string name="drawer_close">Close navigation drawer</string>
 	<string name="drawer_open">Open navigation drawer</string>
 	<string name="default_email">test@liferay.com</string>
 	<string name="default_password">test</string>
+	<string name="forgot_password">Forgot Password?</string>
+	<string name="forgot_password_screenlet_shows_a_form_to_ask_for_a_new_password_using_an_email_address_or_the_username">Forgot Password Screenlet: Shows a form to ask for a new password, using an email address or the username.</string>
```
Comment (phcp) [inline android/forms-screenlet-demo/app/src/main/res/values/strings.xml]: Why too long description? Maybe we should write more real texts for the demo.

File: android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/ForgotPasswordActivity.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		forgotPasswordScreenlet.setAnonymousApiPassword(getString(R.string.liferay_anonymousApiPassword));
+		forgotPasswordScreenlet.setAnonymousApiUserName(getString(R.string.liferay_anonymousApiUserName));
+		forgotPasswordScreenlet.setListener(this);
+	}
+
+	@Override
+	public void onForgotPasswordRequestSuccess(boolean passwordSent) {
+		int backgroundColor =
+			ContextCompat.getColor(this, com.liferay.mobile.screens.viewsets.lexicon.R.color.green_default);
+		int textColor = Color.WHITE;
```
Comment (victorlaerte) [inline android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/ForgotPasswordActivity.java]: Not needed, use Color.WHITE directly

File: android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/LoginActivity.java [hunk 3]
```diff
@@ -36,9 +37,17 @@ protected void onCreate(Bundle savedInstanceState) {
 
 		FormsUtil.setLightStatusBar(this, getWindow());
 
+		View forgotPasswordText = findViewById(R.id.liferay_forgot_link);
```
Comment (victorlaerte) [inline android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/LoginActivity.java]: Use TextView instead View

File: android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/ForgotPasswordActivity.java (hunk 2 above)
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/ForgotPasswordActivity.java]: okay.

File: android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/LoginActivity.java (hunk 3 above)
Comment (camisLuisa) [reply android/forms-screenlet-demo/app/src/main/java/mobile/liferay/com/formsscreenletdemo/LoginActivity.java]: Okay.

**Pull request merged**